cms/static/coffee/spec/views/course_info_spec.coffee

```diff
@@ -232,6 +232,20 @@ define ["js/views/course_info_handout", "js/views/course_info_update", "js/model
 
                 @handoutsEdit.render()
 
+            it "saves <ol></ol> when content left empty", ->
+                requests = AjaxHelpers["requests"](this)
+
```

Comment:
To keep the code consistent. Similar formatting has already been done in this file.